src/v/redpanda/application.cc

```diff
@@ -406,6 +406,14 @@ void application::setup_internal_metrics() {
           sm::description("Redpanda build information"),
           build_labels),
       });
+
+    _metrics.add_group(
```

Comment:
I also feel that this should also be exposed on `/public_metrics`. See `application::setup_public_metrics`.

Reply:
switched to public metrics